Makefile

```diff
@@ -1,6 +1,7 @@
 
-# Image URL to use all building/pushing image targets
+# Image URL to use all building/pushing image targets (IMG will be overwritten during CI operations)
 IMG ?= keikoproj/active-monitor:latest
+LATEST_IMG ?= keikoproj/active-monitor:latest
```

Comment:
the addition of this envVar such that the `latest` tag will always be built/published makes sense to me... but make sure you agree. If we don't want to tag as `latest` on git tag triggered builds, we'd want to change this back

Reply:
If `latest` is essentially pointing to tip then I think it makes sense.

Reply:
ok, thanks for your thoughts

I'm treating `latest` here as the latest built version, regardless of where it came from. Essentially, it means `latest` will be bleeding edge and changing frequently.

as always, the users of AM will need to reference a version tagged docker image (or maybe git hash tagged) for consistency. `latest` won't, and isn't meant to, provide that